hvpy/facade.py

```diff
@@ -0,0 +1,12 @@
+from hvpy.core import execute_api_call
+from hvpy.parameters import getJP2ImageInputParameters
+
+__all__ = ["getJP2Image"]
+
+
+def getJP2Image(**kwargs):
```

Comment:
How should I replace the kwargs

Reply:
For keyword only arguments, see https://peps.python.org/pep-3102/#specification

Reply:
For less wordy info on keyword-only args, see https://docs.python.org/3/glossary.html and search "keyword-only" on that page.